config/checkstyle/suppressions.xml

```diff
@@ -10,8 +10,8 @@
     <suppress checks="Javadoc.*" files=".*Test.java"/>
     <suppress checks="MethodName" files=".*Logger.java"/>
     <suppress checks="MethodName" files=".*LoggerTest.java"/>
-    <suppress checks="MemberName" files="AuthenticationContext.java" lines="813"/>
-    <suppress checks="VisibilityModifier" files="AuthenticationContext.java" lines="813"/>
+    <suppress checks="MemberName" files="AuthenticationContext.java" lines="815"/>
```

Comment:
>815 [](start = 76, length = 3)

this is not scalable, why not use the 
//CHECKSTYLE:OFF: checkstyle:membername

//CHECKSTYLE:ON: checkstyle:membername